### PR TITLE
feat(turbo): g

### PR DESCRIPTION
**Pull request by tknickman**
### Description

Call `@turbo/gen`

Requires https://github.com/vercel/turbo/pull/4872

### CODE REVIEW

File: crates/turborepo-lib/src/cli.rs [hunk 1]
```diff
@@ -264,6 +264,14 @@ pub enum Command {
         #[clap(long, value_enum, default_value_t = LinkTarget::RemoteCache)]
         target: LinkTarget,
     },
+    /// Generate a new app / package
```
Comment (NicholasLYang) [inline crates/turborepo-lib/src/cli.rs:267]: Why G instead of Generate?

File: crates/turborepo-lib/src/commands/generate.rs [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            if result.success() {
+                return Ok(());
+            }
+            return Err(anyhow::anyhow!(ERROR_MSG));
+        }
+        Err(_) => {
+            return Err(anyhow::anyhow!(ERROR_MSG));
+        }
+    }
+}
```
Comment (NicholasLYang) [inline crates/turborepo-lib/src/commands/generate.rs:26]:
```suggestion
    match output {
        Ok(result) if result.success() => Ok(()),
        _ => {
            Err(anyhow::anyhow!("Unable to run generate - missing requirements (npx)"))
        }
    }
```

**Review by NicholasLYang — CHANGES_REQUESTED** [review]

File: crates/turborepo-lib/src/cli.rs (hunk 1 above)
Comment (tknickman) [reply crates/turborepo-lib/src/cli.rs:267]: I'm fine with generate, G was the original name suggested by @nathanhammond so was rolling with it.

Comment (NicholasLYang) [reply crates/turborepo-lib/src/cli.rs:267]: I figured it was a shout out to @rauchg haha

Comment (tknickman) [reply crates/turborepo-lib/src/cli.rs:267]: Well now we definitely need that 😂

Comment (tknickman) [reply crates/turborepo-lib/src/cli.rs:267]: @NicholasLYang updated

```
local-turbo generate --help
Generate a new app / package

Usage: turbo generate [OPTIONS] <COMMAND>

Commands:
  add  Add a new package or app to your project
  run  Run custom generators
```

File: crates/turborepo-lib/src/commands/generate.rs (hunk 2 above)
Comment (NicholasLYang) [reply crates/turborepo-lib/src/commands/generate.rs:26]: Any reason this didn't get applied?

Comment (tknickman) [reply crates/turborepo-lib/src/commands/generate.rs:26]: Oh what....maybe I staged it and didn't apply, thanks for double checking!

**Review by NicholasLYang — APPROVED** [review]

**Pull request merged**